### ISSUE TITLE
Sef Wallet IOS crash

### ISSUE THREAD

**Issue by AboodNASA** [hashmesan/harmony-totp#90]
Hi there
i think the is a problem with the app, I staked my coins with the app and now when I press stake the screen becomes white 
![A2EF0F59-3A70-4341-A334-28276937B9C7](https://user-images.githubusercontent.com/53564842/162369683-82c4b3b7-9a5e-4bb1-8365-dacb4f67a3a8.png)
![56FC2A18-BE64-4D79-A7F0-561BACB7D853](https://user-images.githubusercontent.com/53564842/162369693-e82e5959-2f5e-499f-899d-49d57ca4077f.png)
![5F2A4FBF-2029-4286-9936-36C336AED50E](https://user-images.githubusercontent.com/53564842/162369698-1e679234-eaef-4892-9095-445cbf9c5916.png)

**Comment by hashmesan** [comment]
It could be due to harmony RPC issue. Try reset app and see if that fixes if

**Comment by AboodNASA** [comment]
I tried to reset still the same thing

**Comment by hashmesan** [comment]
Still investigating... Have you seen the staking screen before? What country are you located in?

**Comment by AboodNASA** [comment]
Yes before I stake my tokens i was able to scroll through the validators. I’m located in the UAE

**Comment by hashmesan** [comment]
Can u uninstall and reinstall the app? I assume you wrote the secrets down
so you can import wallet again.
